libcudacxx/include/cuda/std/detail/libcxx/include/version

```diff
@@ -219,6 +219,15 @@ __cpp_lib_void_t                                        201411L <type_traits>
 #pragma GCC system_header
 #endif
 
+// We define certain feature test macros depending on availability. When
+// __CUDA_MINIMUM_ARCH__ is not available, we define the following features
+// unconditionally. This permits checking for the feature in host code. When
+// __CUDA_MINIMUM_ARCH__ is available, we only enable the feature when the
+// hardware supports it.
+#if (!defined(__CUDA_MINIMUM_ARCH__)) || (defined(__CUDA_MINIMUM_ARCH__) && 900 <= __CUDA_MINIMUM_ARCH__)
+# define __cccl_lib_local_barrier_arrive_tx
```

Comment:
If "local" means block-scope, then it should say block scope instead.

Reply:
@griwes @miscco where are all feature macros of cccl documented? 
We should have a page in the docs listing them all, and then this macro needs to be added to that list.

Reply:
This is the first ever feature macro added to libcu++

We recently moved away from hijacking the standard `__cpp_lib_meow` feature test macros and provide `__cccl_lib_meow` instead. We did not yet write documentation for those.

That said I see no requirement to fix this in this PR

Reply:
Local means local shared memory in this case. arrive_tx to remote cluster shared memory has not been implemented due to a dependence on cluster scope barriers, which have not yet been added to the library. 

Resolving this, so discussion can continue in issue #423